<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icons/app_icon.png</normaloff>:/images/icons/app_icon.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="saveStatus">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fontStyle">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
      <property name="acceptRichText">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="wordCount">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fontSize">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Font</string>
    </property>
    <addaction name="action_Decrease_Size"/>
    <addaction name="actionReset_Size"/>
    <addaction name="actionIncrease_Size"/>
    <addaction name="separator"/>
    <addaction name="actionRegular"/>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionBoldItalic"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionLight_Theme"/>
    <addaction name="actionDark_Theme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFont"/>
   <addaction name="menuTheme"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="action_Decrease_Size"/>
   <addaction name="actionReset_Size"/>
   <addaction name="actionIncrease_Size"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionRegular"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionBoldItalic"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionLight_Theme"/>
   <addaction name="actionDark_Theme"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/new.png</normaloff>:/images/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/save.png</normaloff>:/images/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/save_as.png</normaloff>:/images/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save as (Ctrl+Shift+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/copy.png</normaloff>:/images/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy (Ctrl+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/cut.png</normaloff>:/images/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut (Ctrl+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/paste.png</normaloff>:/images/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste (Ctrl+V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/redo.png</normaloff>:/images/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo (Ctrl+Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/undo.png</normaloff>:/images/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo (Ctrl+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/exit.png</normaloff>:/images/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit (Altl+F4)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/about.png</normaloff>:/images/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRegular">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/regular.png</normaloff>:/images/icons/regular.png</iconset>
   </property>
   <property name="text">
    <string>Regular</string>
   </property>
   <property name="toolTip">
    <string>Regular Font (Ctrl+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/bold.png</normaloff>:/images/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Bold Font (Ctrl+B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/italic.png</normaloff>:/images/icons/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic </string>
   </property>
   <property name="toolTip">
    <string>Italic Font (Ctrl+I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBoldItalic">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/bold_italic.png</normaloff>:/images/icons/bold_italic.png</iconset>
   </property>
   <property name="text">
    <string>BoldItalic</string>
   </property>
   <property name="toolTip">
    <string>BoldItalic Font (Ctrl+W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionLight_Theme">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/light_theme.png</normaloff>:/images/icons/light_theme.png</iconset>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
   <property name="toolTip">
    <string>Light Theme</string>
   </property>
  </action>
  <action name="actionDark_Theme">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/dark_theme.png</normaloff>:/images/icons/dark_theme.png</iconset>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
   <property name="toolTip">
    <string>Dark Theme</string>
   </property>
  </action>
  <action name="actionIncrease_Size">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/increase_font_size.png</normaloff>:/images/icons/increase_font_size.png</iconset>
   </property>
   <property name="text">
    <string>Size+</string>
   </property>
   <property name="toolTip">
    <string>Increase font size (Ctrl+Up)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="action_Decrease_Size">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/decrease _font_size.png</normaloff>:/images/icons/decrease _font_size.png</iconset>
   </property>
   <property name="text">
    <string>Size-</string>
   </property>
   <property name="toolTip">
    <string>Decrease font size (Ctrl+Down)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionReset_Size">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/reset_font_size.png</normaloff>:/images/icons/reset_font_size.png</iconset>
   </property>
   <property name="text">
    <string>Reset Size</string>
   </property>
   <property name="toolTip">
    <string>Reset font size (Ctrl+Space)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
